packages/site/src/docs/configurable-imports.md

```diff
@@ -43,7 +45,7 @@ One use case for this functionality is to migrate incrementally from a
 styled-jsx application. When compiling the following file with emotion
 and styled-jsx.
 
-```js
+```jsx
```

Comment:
we should make the `javascript`. It will code highlight in more places.

Reply:
By default Prism only highlights JSX with the jsx language so we would have to change some stuff so IMO we should leave it as jsx.